src/openzaak/components/documenten/models.py

```diff
@@ -317,6 +318,7 @@ class Meta:
         verbose_name = _("Enkelvoudige informatie object")
         verbose_name_plural = _("Enkelvoudige informatie objecten")
         indexes = [models.Index(fields=["canonical", "-versie"])]
+        ordering = ["canonical", "-versie"]
```

Comment:
I'm generally not a fan of this because it messes badly with `GROUP BY` queries, but in this case, I'll allow it. Django is actually working on not using that `ordering` in those queries in the future.